charts/kargo/templates/api/configmap.yaml

```diff
@@ -56,8 +56,8 @@ data:
   {{- if .Values.api.oidc.cliClientID }}
   OIDC_CLI_CLIENT_ID: {{ quote .Values.api.oidc.cliClientID }}
   {{- end }}
-  OIDC_ADDITIONAL_SCOPES: {{ join "," .Values.api.oidc.additionalScopes }}
   {{- end }}
+  OIDC_ADDITIONAL_SCOPES: {{ join "," .Values.api.oidc.additionalScopes }}
```

Comment:
@kpanter thank you! I agree with the change, but would suggest moving this to between existing lines 36 and 37 where I think it will be more obvious that this setting being applied isn't conditional on dex being enabled or not. i.e. It will immediately follow "oidc true". If you don't mind changing that, this will LGTM.

Reply:
@krancour moved OIDC_ADDITIONAL_SCOPES as you suggested.